Add a metaTitle and a metaDescription to every doc pages (#233)
strapi/documentation: docs/user-docs/latest/users-roles-permissions/configuring-administrator-roles.md

---
title: Configure Administrator Roles - Strapi User Guide

> description: Administrators are the users of an admin panel of a Strapi application. Administrator accounts and roles are managed with the Role-Based Access Control (RBAC) feature.

Reviewer: ```suggestion
description: Instructions to configure the administrator roles of a Strapi application with the Role-Based Access Control (RBAC) feature.
```
